R/read_from_redcap-helpers.R

```diff
@@ -23,26 +23,30 @@
 #'   token       = "9A81268476645C4E5F03428B8AC3AA7B",
 #'   records     = c("1", "3", "5"),
 #'   fields      = c("record_id", "name_first", "age", "bmi"),
-#'   id_col_name = NULL,
 #'   id_position = 1
+#'   id_col_name = NULL
 #' )
 #' }
 #' @keywords internal
 #' @noRd
-import_redcap_data <- function(records, fields, uri, token,
-                               id_position, id_col_name) {
+import_redcap_data <- function(uri,
```

Comment:
This should be `base_url` throughout this script and the main redcap script to align with what is done for DHIS2

Reply:
This has been accounted for.